llvm/test/MC/Disassembler/X86/apx/evex-w-opsize.txt

```diff
@@ -0,0 +1,74 @@
+# RUN: llvm-mc -triple x86_64 -disassemble %s | FileCheck %s --check-prefix=ATT
+# RUN: llvm-mc -triple x86_64 -disassemble -output-asm-variant=1 %s | FileCheck %s --check-prefix=INTEL
+
+# This test is to check below inherit:
+# IC_EVEX_W       -->    IC_EVEX_W_OPSIZE
+# IC_EVEX_W_B     -->    IC_EVEX_W_OPSIZE_B
+
```

Comment:
Add tests for NF and NF_ND?

Reply:
We still need to add tests for them since we accept different byte sequence in decoder.

Reply:
I'll add. pls let me know if you want to do that in another commit.

Reply:
It should be added in this commit b/c it satisfy the statement `EVEX.W = 1 and EVEX.pp = 01`

Reply:
7adaff0